src/app/api/services/userService.js

```diff
@@ -16,6 +16,15 @@ const createUser = async (username, email, password) => {
              RETURNING *`,
       [username, email, hash],
     );
+
+  try {
+    const result = await pool.query(
+      `INSERT INTO users(username, email, password_hash)
+             VALUES ($1, $2, crypt($3, gen_salt('md5')))
+             RETURNING *`,
+      [username, email, password],
+    );
+    return result.rows[0];
```

Comment:
this should not have been added back, assume something went wrong with the rebase